generate-new-strings-pot.sh

```diff
@@ -36,36 +26,55 @@ elif [[ "$BRANCH" == "HEAD" ]]; then
 	exit 1
 fi
 
-# bail if msgcat is not installed
-if ! type "msgcat" &> /dev/null; then
-	exit 1
-fi
+function join_by { local d=$1; shift; echo -n "$1"; shift; printf "%s" "${@/#/$d}"; }
+
+# All files changed in this branch
+CHANGED_FILES=$(git diff --name-only $(git merge-base $BRANCH master) $BRANCH '*.js' '*.jsx')
+
+# Diff commits to this branch
+COMMITS_HASHES=$(git log master..$BRANCH --pretty=format:%H);
```

Comment:
I have changed this to simply use the full hash, no real need to use short hashes which entail potential problems when collisions occur.